### PR TITLE
[DNM] Workaround For More Modtimes Bugs

### PR DESCRIPTION
**Pull request by CodaFi**

### CODE REVIEW

**Comment by CodaFi** [comment]
preset=buildbot,tools=RA,stdlib=RD
@swift-ci Please test with preset macOS

File: test/Incremental/CrossModule/external-cascade.swift [hunk 1]
```diff
@@ -29,6 +29,7 @@
 // RUN: touch %t/another-header.h
 // RUN: cd %t && %target-swiftc_driver -c -incremental -emit-dependencies -emit-module -emit-module-path %t/C.swiftmodule -enable-experimental-cross-module-incremental-build -module-name C -I %t -output-file-map %t/C.json -working-directory %t -import-objc-header %t/bridging-header.h -Xfrontend -validate-tbd-against-ir=none -driver-show-incremental -driver-show-job-lifecycle C.swift 2>&1 | %FileCheck -check-prefix MODULE-C %s
 // RUN: cd %t && %target-swiftc_driver -c -incremental -emit-dependencies -emit-module -emit-module-path %t/B.swiftmodule -enable-experimental-cross-module-incremental-build -module-name B -I %t -output-file-map %t/B.json -working-directory %t -driver-show-incremental -driver-show-job-lifecycle B.swift 2>&1 | %FileCheck -check-prefix MODULE-B %s
+// RUN: touch %t/B.swiftmodule
```
Comment (harlanhaskins) [inline test/Incremental/CrossModule/external-cascade.swift:32]: In the module interface tests, we have a `make-old.py` script that sets the modtime to a fixed time in the past. That might be useful here as a source of determinism.

**Comment by CodaFi** [comment]
@swift-ci smoke test

**Comment by CodaFi** [comment]
⛵

**Pull request merged**